Cargo.toml

```diff
@@ -63,7 +63,7 @@ features = [
 
 [target.'cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "openbsd", target_os = "netbsd"))'.dependencies]
 wayland-client = { version = "0.21", features = [ "dlopen", "egl", "cursor"] }
-smithay-client-toolkit = "0.4.3"
+smithay-client-toolkit = { git = "https://github.com/smithay/client-toolkit" }
```

Comment:
Why is the `smithay-client-toolkit` dependency changed to refer to the `git` version instead of the `crates.io` version? I'm pretty sure will prevent us from pushing this update to `crates.io`.

Reply:
This requires changes from SCTK that are not yet released, though given how things are coming, next version of SCTK will be a breaking change (upgrade to wayland-client 0.22).

I plan to make the upgrade as part of the event loop 2.0 refactor, so I'd say this PR should wait until the eventloop 2.0 story is done.

Reply:
Oh wait, that was... written in original PR comment. Whoops. I guess I should read everything before making these sorts of comments :sweat_smile:.